assets/js/base/hooks/cart/use-store-cart.ts

```diff
@@ -7,9 +7,19 @@ import { CART_STORE_KEY as storeKey } from '@woocommerce/block-data';
 import { useSelect } from '@wordpress/data';
 import { useEditorContext } from '@woocommerce/base-context';
 import { decodeEntities } from '@wordpress/html-entities';
-import { mapValues } from 'lodash';
+import type {
+	StoreCart,
+	CartResponseTotals,
+	CartResponseFeeItem,
+	CartResponseBillingAddress,
+	CartResponseShippingAddress,
+} from '@woocommerce/types';
 
-const defaultShippingAddress = {
+declare module '@wordpress/html-entities' {
+	// eslint-disable-next-line no-shadow
+	export function decodeEntities< T >( coupon: T ): T;
+}
```

Comment:
We need this because `decodeEntities` is wrong.
What's there
`(html: string): string` but it actually takes everything and works fine with any type so it should be 
`<T>(html: T): T`